src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -21,7 +21,7 @@ public GlowEnchant(NamespacedKey key) {
     @Override
     @Deprecated
     public String getName() {
-        return "Glow";
+        return "LX_Glow";
```

Comment:
I rather name this GlowEnchant then lx glow

Reply:
Name literally doesn't matter. Pretty sure we don't make it visible and that's likely to conflict with some other plugin

Reply:
Then it looks good to me